update finance with facebooks

stephengroat opened the pull request.

stephengroat committed:
update finance with facebooks

Jawshy commented on _data/finance.yml:
```diff
@@ -193,6 +206,7 @@ websites:
     - name: Shoeboxed
       url: https://www.shoeboxed.com/
       twitter: Shoeboxed
+      facebook: shoeboxed
```
I received this message - including when logged in - when trying to view their Facebook page: 

```
Sorry, this content isn't available at the moment
The link you followed may have expired, or the Page may only be visible to an audience that you aren't in.
Go back to the previous page · Go to News Feed · Visit our Help Centre
```

Is anyone else experiencing this?

stephengroat replied:
i think this, like the `spotify` etc. stuff, is a result of facebook serving pages based off of some sort of geographic information (IP address, facebook profile, etc)

Jawshy commented on _data/finance.yml:
```diff
@@ -208,6 +222,7 @@ websites:
       img: transferwise.png
       tfa: No
       twitter: TransferWise
+      facebook: transferwiseUS
```
As with https://github.com/2factorauth/twofactorauth/pull/2164, what about using non-region specific pages instead? ([transferwise](https://www.facebook.com/transferwise/))

Jawshy committed:
Finance: Update: URL and Facebook

Changes: 
- HelloWallet: URL - No HTTPS support. 
- Mint Bills: Facebook - `mintbills` redirects to `mint`.

stephengroat committed:
Update finance.yml

Jawshy commented on _data/finance.yml:
```diff
@@ -49,14 +54,16 @@ websites:
     - name: InEx Finance
       url: https://www.inexfinance.com/
       twitter: InexFinance
+      facebook: inex.finance
```
I'm not sure about this one. I can't find a link to it from their website. I only end up at [fentury](https://www.facebook.com/fentury).

stephengroat replied:
maybe geo-specific
![screenshot 2016-12-11 at 14 53 44](https://cloud.githubusercontent.com/assets/1159138/21084049/b1760376-bfb1-11e6-8b11-9864923bcd7a.png)

Jawshy reviewed (changes requested):
Everything looks good except this one teensy-tiny lil' thing.

mxxcon commented on _data/finance.yml:
```diff
@@ -49,14 +54,16 @@ websites:
 
     - name: HelloWallet
-      url: https://www.hellowallet.com
+      url: http://www.hellowallet.com
```
Maybe leave it as https in case in the future they won't downgrade? And similar mindset for other similar changes?

Jawshy replied:
Done - https://github.com/2factorauth/twofactorauth/pull/2165/commits/4764ece46eb1ee8946668475a7af42dbd7c03f38.

Jawshy committed:
finance: HelloWallet: Re-add HTTPS

See https://github.com/2factorauth/twofactorauth/pull/2165/files#r92303006
